test/slow/api_spec_slow.js

```diff
@@ -152,6 +152,12 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
       dir = path.resolve(os.tmpdir(), `electron-forge-test-${dirID}/electron-forge-test`);
       dirID += 1;
       await forge.init({ dir });
+
+      const packageJSON = await readPackageJSON(dir);
+      packageJSON.name = 'testApp';
```

Comment:
Here's the error in question:

> MakeAppx : error: Error info: error C00CE169: App manifest validation error: The app manifest must be valid as per schema: Line 6, Column 13, Reason: '' violates minLength constraint of '3'.
> The attribute 'Name' with value '' failed to parse.
> MakeAppx : error: Package creation failed.
> MakeAppx : error: 0x80080204 - The specified package format is not valid: The package manifest is not valid.

How I read that was that it failed because the app name was blank for some reason. But I haven't actually used appx packaging before 😰